### ISSUE TITLE
Missing letters in pdf rendered in webview (WPF)

### ISSUE THREAD

**Issue by andrei88c** [mozilla/pdf.js#12566]
Attach (recommended) or Link to PDF file here:
[test.pdf](https://github.com/mozilla/pdf.js/files/5480295/test.pdf)

Configuration:
- Web browser and its version: Microsoft Edge (EdgeHtml) embedded 
WPF controller:  WebView - "clr-namespace:Microsoft.Toolkit.Wpf.UI.Controls;assembly=Microsoft.Toolkit.Wpf.UI.Controls.WebView"
- Operating system and its version: Win 10 20H2 OS Build 19042.572
- PDF.js version: 2.7.232
- Is a browser extension: No

Steps to reproduce the problem:
sample project: https://github.com/andrei88c/pdfJsTest
1. Run in debug mode.
2. Window opens and renders PDF.

What is the expected behavior? (add screenshot)
Pdf rendered correctly: 
![image](https://user-images.githubusercontent.com/4559333/97968263-169cbc00-1dc7-11eb-8830-f08794df7cf4.png)

What went wrong? (add screenshot)
Missing letters from pdf:
![image](https://user-images.githubusercontent.com/4559333/97968318-29af8c00-1dc7-11eb-93b1-60d1b63be0d8.png)

**Comment by Snuffleupagus** [comment]
Looks like a duplicate of #12462; note https://github.com/mozilla/pdf.js/issues/12462#issuecomment-713878908 in particular.

**Comment by andrei88c** [comment]
Waiting on the bug fix to test on this implementation.

**Comment by timvandermeij** [comment]
Closing as a duplicate of that issue for now.
